packages/perseus/src/widgets/categorizer/categorizer.tsx

```diff
@@ -92,8 +92,12 @@ export class Categorizer
         const isMobile = this.props.apiOptions.isMobile;
         let indexedItems = this.props.items.map((item, n) => [item, n]);
         if (this.props.randomizeItems) {
-            // @ts-expect-error - TS4104 - The type 'readonly (string | number)[][]' is 'readonly' and cannot be assigned to the mutable type '(string | number)[][]'. | TS2345 - Argument of type 'number | null | undefined' is not assignable to parameter of type 'number | RNG'.
-            indexedItems = Util.shuffle(indexedItems, this.props.problemNum);
+            // @ts-expect-error - TS4104 - The type 'readonly (string | number)[][]' is 'readonly' and cannot be assigned to the mutable type '(string | number)[][]'.
+            indexedItems = CoreUtil.shuffle(
+                indexedItems,
+                // @ts-expect-error - TS2345 - Argument of type 'number | null | undefined' is not assignable to parameter of type 'number | RNG'.
+                this.props.problemNum,
```

Comment:
Instead of a `ts-expect-error`, could you use a cast to `any`? Polaris work is going to disallow `ts-expect-error` for many reasons. Casting to `any` should suppress the error but is not quite so risky. 

```suggestion
                this.props.problemNum as any,
```

Reply:
Removed the expect error comment and cast it to any :)